src/__experimental__/components/textbox/textbox.d.ts

```diff
@@ -1,6 +1,7 @@
 import * as React from "react";
 import { SpaceProps } from "styled-system";
 import { ValidationPropTypes } from "../../../components/validations";
+import * as OptionsHelper from "../../../utils/helpers/options-helper";
```

Comment:
OptionsHelper is soon to be removed by this PR https://github.com/Sage/carbon/pull/4134
I recommend that we delay merging this PR until #4134 is merged, then we'll advise on next steps for you